public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/route-settings/RouteSettings.tsx

```diff
@@ -97,8 +94,20 @@ export const RoutingSettings = ({ alertManager }: RoutingSettingsProps) => {
                     // @ts-ignore-check: react-hook-form made me do this
                     setValue(`contactPoints.${alertManager}.groupBy`, [...field.value, opt]);
                   }}
-                  onChange={(value) => onChange(mapMultiSelectValueToStrings(value))}
+                  onChange={(value) => {
+                    return onChange(mapMultiSelectValueToStrings(value));
+                  }}
                   options={[...commonGroupByOptions, ...groupByOptions]}
+                  components={{
+                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                    MultiValueRemove(props: any) {
```

Comment:
Why can't we get proper types?

Reply:
I couldn't find a proper types in here, let me check it again

Reply:
I found this could work: 

```
  MultiValueRemove(props: React.PropsWithChildren<MultiValueRemoveProps & Array<SelectableValue<string>> & ({
      data: {
        label: string;
        value: string;
        isFixed: boolean;
      }
    })>)
```